x-pack/legacy/plugins/siem/public/pages/case/components/open_closed_stats/index.tsx

```diff
@@ -12,19 +12,28 @@ export interface Props {
   caseCount: number | null;
   caseStatus: 'open' | 'closed';
   isLoading: boolean;
+  dataTestSubj?: string;
 }
 
-export const OpenClosedStats = React.memo<Props>(({ caseCount, caseStatus, isLoading }) => {
-  const openClosedStats = useMemo(
-    () => [
-      {
-        title: caseStatus === 'open' ? i18n.OPEN_CASES : i18n.CLOSED_CASES,
-        description: isLoading ? <EuiLoadingSpinner /> : caseCount ?? 'N/A',
-      },
-    ],
-    [caseCount, caseStatus, isLoading]
-  );
-  return <EuiDescriptionList textStyle="reverse" listItems={openClosedStats} />;
-});
+export const OpenClosedStats = React.memo<Props>(
+  ({ caseCount, caseStatus, isLoading, dataTestSubj }) => {
+    const openClosedStats = useMemo(
+      () => [
+        {
+          title: caseStatus === 'open' ? i18n.OPEN_CASES : i18n.CLOSED_CASES,
+          description: isLoading ? <EuiLoadingSpinner /> : caseCount ?? 'N/A',
+        },
+      ],
+      [caseCount, caseStatus, isLoading, dataTestSubj]
+    );
+    return (
+      <EuiDescriptionList
+        data-test-subj={dataTestSubj}
```

Comment:
Same here :)

Reply:
on these ones where its not on the object, you can default it on line i18 by doings `dataTestSubj = ''`

Reply:
ok! thx :)

Reply:
@stephmilovic https://github.com/elastic/kibana/pull/63971#discussion_r412014352. What do you think?

Reply:
yep thats easy enough